scripts/aws/aws_helper.py

```diff
@@ -13,6 +13,7 @@
     Bucket = object
 
 Credentials = namedtuple("Credentials", ["access_key", "secret_key", "token"])
+S3_path = namedtuple("S3_path", ["bucket", "key"])
```

Comment:
Why named tuples? do you have reasoning for using them vs other container classes?

Also mixing Snake and TitleCase is a little odd,  what is the prefered method for naming Objects Title?

Reply:
> Why named tuples? do you have reasoning for using them vs other container classes?

1. I was using a `tuple` before and did not want to create a `class` when the purpose was only to name those properties.
2. I found this interesting topic, https://stackoverflow.com/questions/51671699/data-classes-vs-typing-namedtuple-primary-use-cases

> Also mixing Snake and TitleCase is a little odd, what is the prefered method for naming Objects Title?

This is purely a mistake, this should be `s3_path`, good spotting.

Reply:
Just looking at the articile you may want `NamedTuple` then

```python
from collections import namedtuple
from typing import NamedTuple

Foo = namedtuple("Foo", ["a", "b"])
Foo("a","b").a # Any
Foo("a", 1).b # Any

Bar = NamedTuple("Bar", [("a", str), ("b", int)])
Bar("a", 1).a # str
Bar("a", 1).b # int
```

Reply:
I should specify the type and use TitleCase then, if it is the convention